keras_core/trainers/trainer.py

```diff
@@ -34,6 +34,92 @@ def compile(
+                instance. See `keras_core.metrics`. Typically you will use
+                `metrics=['accuracy']`. A function is any callable with the
+                signature `result = fn(y_true, _pred)`. To specify different
+                metrics for different outputs of a multi-output model, you could
+                also pass a dictionary, such as
+                `metrics={'a':'accuracy', 'b':['accuracy', 'mse']}`.
+                You can also pass a list to specify a metric or a list of
+                metrics for each output, such as
+                `metrics=[['accuracy'], ['accuracy', 'mse']]`
+                or `metrics=['accuracy', ['accuracy', 'mse']]`. When you pass
+                the strings 'accuracy' or 'acc', we convert this to one of
+                `keras_core.metrics.BinaryAccuracy`,
+                `keras_core.metrics.CategoricalAccuracy`,
+                `keras_core.metrics.SparseCategoricalAccuracy` based on the
+                shapes of the targets and of the model output. We do a similar
+                conversion for the strings 'crossentropy' and 'ce' as well.
+                The metrics passed here are evaluated without sample weighting;
+                if you would like sample weighting to apply, you can specify
+                your metrics via the `weighted_metrics` argument instead.
+            weighted_metrics: List of metrics to be evaluated and weighted by
+                `sample_weight` or `class_weight` during training and testing.
+            run_eagerly: Bool. If `True`, this `Model`'s logic will never be
+                compiled (e.g. with `tf.function` or `jax.jit`). Recommended to
+                leave this as `False` when training for best performance, and
+                `True` when debugging.
+            steps_per_execution: Int. The number of batches to run
+                during each a single compiled function call. Running multiple
+                batches inside a single a single compiled function call can
+                greatly improve performance on TPUs or small models with a large
+                Python overhead. At most, one full epoch will be run each
+                execution. If a number larger than the size of the epoch is
+                passed, the execution will be truncated to the size of the
+                epoch. Note that if `steps_per_execution` is set to `N`,
+                `Callback.on_batch_begin` and `Callback.on_batch_end` methods
+                will only be called every `N` batches (i.e. before/after
+                each compiled function execution).
+            jit_compile: Bool or `"auto"`. Whether to use XLA compilation when
+                compiling a model. This value should currently never be `True`
+                on the torch backed, and should always be `True` or `"auto"` on
+                the jax backend. On tensorflow, this value can be `True` or
```

Comment:
This is not entirely accurate. You can pass `jit_compile=False` with JAX, which means that the model will train/evaluate/predict in eager execution (so `jit_compile` is equivalent to `not run_eagerly`). TF is special in the sense that it supports the unique combination `jit_compile == False and run_eagerly == False`, which means "classic graph mode".